.github/workflows/ci.yml

```diff
@@ -100,6 +100,39 @@ jobs:
           KERNELDIR=/lib/modules/$(ls /lib/modules)/build make -j4
           make run-unit-tests
 
+  build-shared-libs-linux-amd64:
+    name: build-shared-libs-linux-amd64 🧐
+    runs-on: ubuntu-latest
+    container:
```

Comment:
I think we might want to drop the container, so that the job runs on the ubuntu-latest host, which we are installing the headers against.
This should finally fix the ci, i think!

Reply:
Done. Thanks!